rerun_py/rerun_sdk/rerun/log_deprecated/rects.py

```diff
@@ -180,16 +180,16 @@ def log_rects(
         half_sizes = rects[:, 2:4] / 2
         centers = rects[:, 0:2] + half_sizes
     elif rect_format == RectFormat.YXHW:
-        half_sizes = rects[:, 4:2] / 2
-        centers = rects[:, 2:0] + half_sizes
```

Comment:
`i0 >= i1` will always return an empty array no matter what... we just never tested this path.